server/src/routes/v2/apps.js

```diff
@@ -1,24 +1,67 @@
-const Boom = require('@hapi/boom')
-const Joi = require('@hapi/joi')
+const AppModel = require('../../models/v1/out/App')
+const { AppStatus } = require('../../enums')
+const { getApps } = require('../../data')
+const { convertAppsToApiV1Format } = require('../v1/apps/formatting')
+const { filterAppsBySpecificDhis2Version } = require('../../utils/filters')
+const Joi = require('../../utils/CustomJoi')
 
-const AppModel = require('../../models/v2/out/App')
+const CHANNELS = ['Stable', 'Development', 'Canary']
+const APPTYPES = ['APP', 'DASHBOARD_WIDGET', 'TRACKER_DASHBOARD_WIDGET']
```

Comment:
Similarly for app types, but capitalized snake casing is a little more normalized - in any case we should be consistent

Reply:
I think the app types are named that way to reflect [how `dhis2-core` stores them](https://github.com/dhis2/dhis2-core/blob/ea76fa86409613c9766d4508d65c88ac55c413c3/dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/AppType.java). Is there a risk that renaming them would impact core (through its use of v1 APIs)? I guess we could convert e.g. `dashboardWidget` to `DASHBOARD_WIDGET` in v1 responses

Reply:
I think we should keep these consistant with the dhis2-core naming.

Reply:
Sounds good to me, better to be safe and consistent